utils/build-windows-toolchain.bat

```diff
@@ -68,6 +68,10 @@ if "%TestArg:~-1%"=="," (set TestArg=%TestArg:~0,-1%) else (set TestArg= )
 set SkipPackagingArg=-SkipPackaging
 if not "%SKIP_PACKAGING%"=="1" set "SkipPackagingArg= "
 
+:: Build the -WindowsSDKs argument, if any
```

Comment:
This seems odd. The default behaviour (without the flag) is to build all the SDKs (which is intentional as we have caught platform specific issues this way). Adding in the filter is fine, but I think that this is better reflected in the comment as: "Build only the specified SDKs, defaulting to all if unspecified.".